client/my-sites/people/people-invites/index.jsx

```diff
@@ -204,7 +204,7 @@ class PeopleInvites extends React.PureComponent {
 		return (
 			<React.Fragment>
 				<Button busy={ deleting } compact onClick={ this.toggleClearAllConfirmation }>
-					{ translate( 'Clear All Accepted' ) }
+					{ translate( 'Clear all accepted' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.